entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -l
+
+image=$1
+report=$2
+patched_tag=$3
+
+# parse image into image name and image tag
+image_no_tag=$(echo "$image" | cut -d':' -f1)
+old_tag=$(echo "$image" | cut -d':' -f2)
+
+# new patched image tag
+new_tag="$old_tag$patched_tag"
+
+# run copa to patch image
+if copa patch -i "$image" -r /data/"$report" -t "$new_tag" --addr tcp://0.0.0.0:8888;
+then
+    patched_image="$image_no_tag:$new_tag"
+    echo "::set-output name=patched-image::$patched_image"
```

Comment:
this is deprecated 
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
looking into this, running into issues with GITHUB_OUTPUT